packages/core/src/controllers/events.ts

```diff
@@ -210,13 +211,21 @@ export class EventClient extends IEventClient {
   };
 
   private sendEvent = async (events: EventClientTypes.Event[]) => {
+    // if domain isn't available, set `sp` as `desktop` so data would be extracted on api side
+    const platform = this.getAppDomain() ? "" : "desktop";
```

Comment:
You can have only one ternary and then use this variable on line 219
```suggestion
    const platformQueryParam = this.getAppDomain() ? "" : "&sp=desktop";
```

Reply:
yes, requests are sent. Updating